azure-pipelines.yml

```diff
@@ -36,10 +36,28 @@ jobs:
   - script: dotnet restore -s https://api.nuget.org/v3/index.json
   - script: bash scripts/test.sh --filter
 
+- job: windows_2016_vs2017_npcap
```

Comment:
Visual Studio version have little to no importance in test cases, what matters really is the Windows version and driver (Npcap or WinPcap)

Reply:
@kayoub5 agree but also want to make sure we are testing what people are normally using. It's easy enough to add the vs2017 and older windows builds. Not sure there is a big downside.